### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by AlexanderHolmeset**
It should always be a recomendation to use MFA.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you're a user, you don't have the permissions to set your password to never e
    
 As an admin, you can make user passwords expire after a certain number of days, or set passwords to never expire. 
 > [!Tip]
-> By default, passwords are set to never expire. Current research strongly indicates that mandated password changes do more harm than good. They drive users to choose weaker passwords, re-use passwords, or update old passwords in ways that are easily guessed by hackers.  
+> By default, passwords are set to never expire. Current research strongly indicates that mandated password changes do more harm than good. They drive users to choose weaker passwords, re-use passwords, or update old passwords in ways that are easily guessed by hackers. If setting password to never expire its recommended to enable [multi-factor authentication](../security-and-compliance/set-up-multi-factor-authentication.md).
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/set-password-expiration-policy.md]:
```suggestion
> By default, passwords are set to never expire. Current research strongly indicates that mandated password changes do more harm than good. They drive users to choose weaker passwords, re-use passwords, or update old passwords in ways that are easily guessed by hackers. If setting password to never expire, we recommend enabling [multi-factor authentication](../security-and-compliance/set-up-multi-factor-authentication.md).
```

**Comment by mypil** [comment]
@AlexanderHolmeset  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kwekuako - PR has been copy edited and is ready for final review.  

Thank you.

cc: @kenwith

**Pull request merged**